### ISSUE TITLE
CMake's FindRuby  on Windows can't find the ruby 2.7 installed by conda-forge

### ISSUE THREAD

**Issue by traversaro** [conda-forge/ruby-feedstock#43]
The upstream CMake issue is https://gitlab.kitware.com/cmake/cmake/-/issues/21066, but I wanted to open an issue also here for visibility. The problem is that FindRuby has not been updated with the latest possible names of the Ruby library. In my case for example, I have:
~~~
-- _Ruby_POSSIBLE_LIB_NAMES: ruby;ruby-static;ruby2.7;ruby27;ruby-2.7;ruby-2.7.2;x64-msvcr141-ruby272;x64-msvcr141-ruby272-static;x64-msvcrt-ruby272;x64-msvcrt-ruby272-static
~~~
bu the library is actually called `x64-vcruntime140-ruby270` .

**Comment by traversaro** [comment]
CMake PR: https://gitlab.kitware.com/cmake/cmake/-/merge_requests/5486 .

**Comment by traversaro** [comment]
> CMake PR: https://gitlab.kitware.com/cmake/cmake/-/merge_requests/5486 .

At the fix is for an environmental regression, the change has been target toward the 3.19 release branch, and so should be part of the next 3.19.0 CMake release (that I guess is going to happen in November).

**Comment by traversaro** [comment]
> 
> 
> > CMake PR: https://gitlab.kitware.com/cmake/cmake/-/merge_requests/5486 .
> 
> At the fix is for an environmental regression, the change has been target toward the 3.19 release branch, and so should be part of the next 3.19.0 CMake release (that I guess is going to happen in November).

PR on CMake merged, let's wait for the CMake 3.19 release and its adoption in conda-forge to close the issue.

**Comment by traversaro** [comment]
The fix was released as part of CMake 3.19.0 : https://discourse.cmake.org/t/cmake-3-19-0-available-for-download/2198 , so we will fix the issue once CMake 3.19 is released in conda-forge . However CMake 3.19.0 contains a quite critical regression (it breaks for example LLVM, see https://gitlab.kitware.com/cmake/cmake/-/issues/21469) so it may be better to just wait for CMake 3.19.1 release.
